### PR TITLE
Added hanlding for fs exception

### PR DESCRIPTION
**Pull request by jimni1222**
## Proposed changes

This PR introduces adding exception handling for web browser.

## Types of changes

Please put an x in the boxes related to your change.

- [x] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-js/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-js)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/caver-ipfs/src/index.js [hunk 1]
```diff
@@ -53,19 +53,55 @@ class IPFS {
...
+     * module.exports = {
+     *     ...
+     *     node: {
+     *         fs: 'empty',
+     *     },
+     *     ...
+     * }
+     * @returns {null}
+     *
+     * If you use next web framework, add the following to your "next.config.json" file.
```
Comment (jimni1222) [inline packages/caver-ipfs/src/index.js:73]:
```suggestion
     * If you use next-js web framework, add the following to your "next.config.json" file.
```

File: packages/caver-ipfs/src/index.js [hunk 2]
```diff
@@ -53,19 +53,55 @@ class IPFS {
...
+     * module.exports = {
+     *     ...
+     *     node: {
+     *         fs: 'empty',
+     *     },
+     *     ...
+     * }
+     * @returns {null}
+     *
+     * If you use next-js web framework, add the following to your "next.config.json" file.
```
Comment (jimni1222) [inline packages/caver-ipfs/src/index.js:73]:
```suggestion
     * If you use Next.js web framework, add the following to your "next.config.json" file.
```

File: packages/caver-ipfs/src/index.js [hunk 3]
```diff
@@ -53,19 +53,55 @@ class IPFS {
...
     /**
-     * adds a file to IPFS
+     * adds a file to IPFS.
+     * If the `data` parameter is a `Buffer` or `ArrayBuffer`, upload to IPFS directly without using `fs`.
+     * If the `data` parameter is a string, use `fs` to read the file.
+     * Since `fs` is a module that can only be used on the server side, if it is client-side code,
+     * it must read the file in advance and pass the file contents in the format of `ArrayBuffer`.
      *
-     * @param {string|Buffer} data The file path string or file contents.
+     * If you get a "Error: Can't resolve'fs'" error when building your client code, add the following to your "webpack.config.json" file.
```
Comment (sirano11) [inline packages/caver-ipfs/src/index.js:62]:
```suggestion
     * If you get a "Error: Can't resolve 'fs'" error when building your client code, add the following to your "webpack.config.json" file.
```

File: packages/caver-ipfs/src/index.js (hunk 1 above)
Comment (sirano11) [inline packages/caver-ipfs/src/index.js]:
```suggestion
     * If you use next web framework(https://nextjs.org/), add the following to your "next.config.json" file.
```

File: packages/caver-ipfs/src/index.js [hunk 4]
```diff
@@ -53,19 +53,55 @@ class IPFS {
...
+     * module.exports = {
+     *     ...
+     *     node: {
+     *         fs: 'empty',
+     *     },
+     *     ...
+     * }
+     * @returns {null}
+     *
+     * If you use Next.js web framework, add the following to your "next.config.json" file.
```
Comment (jimni1222) [inline packages/caver-ipfs/src/index.js:73]:
```suggestion
     * If you use Next.js web framework(https://nextjs.org/), add the following to your "next.config.json" file.
```

File: packages/caver-ipfs/src/index.js (hunk 1 above)
Comment (jimni1222) [reply packages/caver-ipfs/src/index.js]: This is outdated, so i applied separately.

**Review by sirano11 — APPROVED** [review]

**Pull request merged**